uint8 id_module
uint8 submodule
uint8 status